Import css/css-conditional/container-queries and sync css/css-contain from WPT

karlcow opened the pull request:
#### b44aca25e295bfa16359d394b0d2ab87d3b3e938
<pre>
Import css/css-conditional/container-queries and sync css/css-contain from WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=277126">https://bugs.webkit.org/show_bug.cgi?id=277126</a>
<a href="https://rdar.apple.com/132554228">rdar://132554228</a>

Reviewed by Tim Nguyen.

First import of css/css-conditional/container-queries/
Re-Synchronization of css/css-contain/

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/bf8abb12d477272d8169275d99c95b04526e15a0">https://github.com/web-platform-tests/wpt/commit/bf8abb12d477272d8169275d99c95b04526e15a0</a>

This will also help with <a href="https://webkit.org/b/277122">https://webkit.org/b/277122</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/container-queries/*
  Adding new directory of WPT tests including their expected.txt

* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/*
  Resynchronizing css-contain which had a lot of duplicates with container-queries

* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/281464@main">https://commits.webkit.org/281464@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a12ceb1118d40ecce733b2a3c83f20c5dff68650

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/60007 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/39355 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/12559 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/63925 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/10537 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/47027 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/10744 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/48623 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/7353 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/62037 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/36698 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/51962 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/29465 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/33403 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/9209 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/9457 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/55316 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/9487 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/65657 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/3937 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/62/builds/9350 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/55974 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/3955 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/51955 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/56124 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/13298 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/3268 "Passed tests") | | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/35168 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/36250 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/37338 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/35994 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (commented):
Can you also resync css-contain in the same commit? Many of these tests are now just duplicates of css-contain/container-queries

karlcow committed:
Import css/css-conditional/container-queries and sync css/css-contain from WPT
https://bugs.webkit.org/show_bug.cgi?id=277126
rdar://132554228

Reviewed by Tim Nguyen.

First import of css/css-conditional/container-queries/
Re-Synchronization of css/css-contain/

Upstream commit: https://github.com/web-platform-tests/wpt/commit/bf8abb12d477272d8169275d99c95b04526e15a0

This will also help with https://webkit.org/b/277122

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/container-queries/*
  Adding new directory of WPT tests including their expected.txt

* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/*
  Resynchronizing css-contain which had a lot of duplicates with container-queries

* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/281464@main
